projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -149,8 +149,14 @@ build-ami-ubuntu-2004: setup-ami-share deps-ami setup-packer-configs-ami
 release-ami-ubuntu-2004: MAKEFLAGS=
 release-ami-ubuntu-2004: FULL_OUTPUT_DIR=$(MAKE_ROOT)/$(OUTPUT_DIR)/ami
 release-ami-ubuntu-2004: PACKER_TYPE_VAR_FILES=$(PACKER_AMI_VAR_FILES)
+release-ami-ubuntu-2004: export MANIFEST_OUTPUT=$(FULL_OUTPUT_DIR)/manifest.json
+release-ami-ubuntu-2004: EXPORT_AMI_BUCKET?=$(ARTIFACTS_BUCKET)
+release-ami-ubuntu-2004: AMI_S3_DST=$(EXPORT_AMI_BUCKET)/$(ARTIFACTS_UPLOAD_PATH)/ami
+release-ami-ubuntu-2004: EXPORT_AMI_DST=$(AMI_S3_DST)/$(GIT_HASH)
```

Comment:
I cant tell for sure, but just to make sure.  What we usually do is upload artifacts the GIT_HASH folder for the specific build and then also to "latest" to make it easier to pull from other projects and what not.  Is this uploading to both spots?

Reply:
Yeah
The `image-export` will "upload" it to the `GIT_HASH` folder
And then we have an extra `s3 cp` command that copies it from there to `latest`

Reply:
ok cool thats what i thought but wanted to be sure.